Modules/Filtering/Smoothing/test/itkFFTDiscreteGaussianImageFilterTest.cxx

```diff
@@ -95,6 +95,16 @@ itkFFTDiscreteGaussianImageFilterTestProcedure(int argc, char ** argv)
 
   itk::WriteImage(filter->GetOutput(), argv[3], true);
 
+  // Test streaming enumeration for FFTDiscreteGaussianImageFilterEnums::KernelSource elements
+  const std::set<itk::FFTDiscreteGaussianImageFilterEnums::KernelSource> allKernelSource{
+    itk::FFTDiscreteGaussianImageFilterEnums::KernelSource::OPERATORS,
+    itk::FFTDiscreteGaussianImageFilterEnums::KernelSource::IMAGE_SOURCE
+  };
+  for (const auto & ee : allKernelSource)
```

Comment:
What was the reason to construct an `std::set` of these enum values? I saw it before, I'm just wondering why. FYI, the following also works, creating an initializer-list "on the fly": 

    for (const auto enumValue : { 
        itk::FFTDiscreteGaussianImageFilterEnums::KernelSource::OPERATORS,
        itk::FFTDiscreteGaussianImageFilterEnums::KernelSource::IMAGE_SOURCE} )
    {
        ....
    }

In general, enum values do not need to be passed by "const reference" (`const auto &`), because they can be copied quickly, so I would rather leave the ampersand `&` out.

Just my two cents. HTH, Niels

Reply:
I based my changes off of a previous implementation in ITK, I like yours better 😃 Updated.